yarn-project/prover-node/src/prover-node.test.ts

```diff
@@ -141,13 +148,7 @@ describe('prover-node', () => {
     quoteSigner.sign.mockImplementation(payload => Promise.resolve(new EpochProofQuote(payload, Signature.empty())));
 
     // We create 3 fake blocks with 1 tx effect each
-    blocks = times(3, i =>
-      mock<L2Block>({
-        number: i + 20,
-        hash: () => new Fr(i),
-        body: mock<Body>({ txEffects: [mock<TxEffect>({ txHash: TxHash.random() } as TxEffect)] }),
```

Comment:
This took longer than I want to admit to debug: mock makes the inner instances show up as undefined